app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
```

Comment:
On the current implementation the tasks page is the default index page, we could use that controller instead